src/components/media-player/media-player-progress-bar.less

```diff
@@ -58,7 +56,7 @@
   justify-content: center;
   flex-direction: column;
   z-index: @edu-z-index-overlay-layer;
-  top: -@edu-progress-bar-tooltip-height - 10px;
+  top: -40px;
```

Comment:
There just needed to be more space between the tooltip and the tooltip trigger (aka the bar) and that space cannot be calculated in CSS because it has to consider the height of the tooltip itself.